apps/taxes/src/modules/avatax/ui/avatax-configuration-credentials-fragment.tsx

```diff
@@ -121,7 +121,17 @@ export const AvataxConfigurationCredentialsFragment = (
             }
             name="isSandbox"
           />
-
+          <AppToggle
+            control={control}
+            label="Document recording"
+            helperText={
+              <HelperText>
+                When turned off, the document type will always be set to <i>SalesOrder</i>. This
+                means the transactions will not be recorded in Avatax.
+              </HelperText>
```

Comment:
here also lets add docs (https://developer.avalara.com/erp-integration-guide/sales-tax-badge/design-document-workflow/option-to-disable-document-recording/)

we should do that everywhere we can